packages/db/src/proxy.ts

```diff
@@ -133,6 +133,30 @@ function deepClone<T extends unknown>(
     return clone as unknown as T
   }
 
+  // Handle Temporal objects
+  // Check if it's a Temporal object by checking for the Temporal brand
+  if (typeof (obj as any)[Symbol.toStringTag] === `string`) {
+    const tag = (obj as any)[Symbol.toStringTag]
+    const temporalTypes = [
+      `Temporal.PlainDate`,
+      `Temporal.PlainTime`,
+      `Temporal.PlainDateTime`,
+      `Temporal.ZonedDateTime`,
+      `Temporal.Instant`,
+      `Temporal.PlainYearMonth`,
+      `Temporal.PlainMonthDay`,
+      `Temporal.Duration`,
+      `Temporal.TimeZone`,
```

Comment:
These 2 temporals (`TimeZone` and `Calendar`) don't seem to exist according to the docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal